tsdb/db.go

```diff
@@ -201,7 +201,7 @@ func newDBMetrics(db *DB, r prometheus.Registerer) *dbMetrics {
 		Help: "Size of symbol table in memory for loaded blocks",
 	}, func() float64 {
 		db.mtx.RLock()
-		blocks := db.blocks[:]
+		blocks := db.blocks
```

Comment:
This code is not doing the same thing, as it's now longer returning a different slice.